app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/Formatters.scala

```diff
@@ -47,21 +54,19 @@ trait Formatters {
 
   private[mappings] def booleanFormatter(
     requiredKey: String,
-    invalidKey: String,
-    args: Seq[String] = Seq.empty,
-    sanitise: String => String
+    invalidKey: String
   ): Formatter[Boolean] =
     new Formatter[Boolean] {
 
-      private val baseFormatter = stringFormatter(requiredKey, args, sanitise)
+      private val baseFormatter = stringFormatter(requiredKey, removeAllWhitespace = true)
 
       override def bind(key: String, data: Map[String, String]) =
         baseFormatter
           .bind(key, data)
           .flatMap {
             case "true"  => Right(true)
             case "false" => Right(false)
-            case _       => Left(Seq(FormError(key, invalidKey, args)))
+            case _       => Left(Seq(FormError(key, invalidKey)))
```

Comment:
Why remove `args`?

Reply:
Because it is not being used for anything, so if it's not needed then it can be removed